training/general/src/environments/acpt-pytorch-1.11-cuda11.5/spec.yaml

```diff
@@ -14,6 +13,8 @@ build:
 os_type: linux
 
 tags:
+  Deprecated: ""
+  DeleteOn: "2023-04-14"
```

Comment:
We are not deleting it. We do not support metadata update yet (i.e. archive)

Reply:
ok, so just keep the deprecate tag?

Reply:
Also how do we make sure, its removed from vulnerabilities?

Reply:
thanks @elesel . I have removed the tag 'DeleteOn'